cspell-custom-words.txt

```diff
@@ -0,0 +1,194 @@
+Acala
+acala
+Acks
+Alertmanager
+algod
+Algorand
+algosdk
+alist
+Aptos
+authorisation
+authorise
+authorised
+backdoors
+Bech
+bech
+behaviour
+bigset
+Bigtable
+borsh
+bscscan
+BUILDKIT
+bytecodes
+callstack
+CCTP
+Celestia
+celestia
+certusone
+Chainlink
+Coinspect
+collateralization
+colour
+Concat
+conftest
+Cosm
+cosmoshub
+Cosmwasm
+cosmwasm
+counterparty
+cpus
+crosschain
+Cyfrin
+datagram
+denoms
+devnet
+Dymension
+dymension
+EMMITTER
+ethcrypto
+ethersproject
+ETHRPC
+evmos
+forgeable
+FQTs
+frontends
+funder
+gitub
+gogoproto
+goimports
+gossipv
+GUARDIAND
+guardiand
+guardiand's
+Hacken
+hashdump
+hexdump
+Holesky
+holesky
+ICCO
+incentivized
+incentivizing
+initialisation
+initialised
+initialiser
+Injective
+injective
+inotify
+intcblock
+IPFS
+journalctl
+Karura
+karura
+Keccak
+KEVM
+kevm
+keymap
+keytool
+Klaytn
+klaytn
+kompiled
+kompiles
+Kudelski
+Kujira
+kujira
+lamports
+lastrun
+libp
+localterra
+lockfiles
+merkle
+Metaplex
+mindshare
+mlock
+moby
+moonscan
+moretags
+nearapi
+Neodyme
+Newtypes
```

Comment:
Maybe this should be removed. I only see it as the section header for a TODO in the Aptos readme.

Reply:
Let me know what you think of ee0858e36ff097bed342a353cafbe5393c0d0ce8

I see:
```shell
$ cspell '**/*.md' --config cspell.config.yaml --words-only --unique --no-issues --no-progress
CSpell: Files checked: 90, Issues found: 0 in 0 files.
```